.github/workflows/test_docs_build.yml

```
@@ -0,0 +1,56 @@
+name: Documentation
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+      - develop
+  schedule:
+    # Run every day at 3 am UTC
+    - cron: "0 3 * * *"
```

I would say we can simply this
```suggestion
on:
  workflow_dispatch:
  pull_request:
```

---

Understood  Eric
, is there any further change ?

---

I saw one more thing that was missed from Agriya's review,  but I think that is it from my side